### ISSUE TITLE
Could not execute macro drawgraph inside cpu page

### ISSUE THREAD

**Issue by john-kheir** [0-complexity/openvcloud#1409]
#### Detailed description
![error](https://user-images.githubusercontent.com/15011431/36943820-b8b5acb6-1f98-11e8-806e-43c38b59e14f.png)

#### Installation information
JumpScale

    Core: branch: master (0fdeaf3) 3/2/2018, 8:41:51 PM
    Portal: branch: master (3df0a29) 3/1/2018, 8:00:47 PM

OpenvCloud

    Core: branch: master (11f0302) 3/4/2018, 9:20:46 AM
    G8VDC: branch: master (d4b49c1) 2/13/2018, 1:56:43 PM
    Selfhealing: branch: master (8f2a952) 3/1/2018, 5:40:22 PM

OpenvStorage be-g8-3

    openvstorage-backend-core: 1.9.2-1
    openvstorage-health-check: 3.4.0-1
    openvstorage-webapps: 2.9.12-1
    openvstorage-backend-webapps: 1.9.2-1
    openvstorage-core: 2.9.12-1
    openvstorage-hc: 1.9.2-1
    alba-ee: 1.5.26
    openvstorage: 2.9.12-1
    openvstorage-backend: 1.9.2-1
    openvstorage-extensions: 0.1.2-1
    openvstorage-sdm: 1.9.1-1

**Comment by abdulrahmantkhalifa** [comment]
fixed on openvcloud_installer 5fd8cc0e4636798f4d0640405f09609f9fc8bf9c

**Comment by Dinaamagdy** [comment]
## Software Version 
```
JumpScale
Core: branch: master (0de3a2b) 3/4/2018, 2:06:10 PM
Portal: branch: master (3bb6e8e) 3/4/2018, 6:44:27 PM
OpenvCloud
Core: branch: master (7e570f3) 3/4/2018, 8:49:52 PM
G8VDC: branch: master (d4b49c1) 2/13/2018, 1:56:43 PM
Selfhealing: branch: master (4adf360) 3/4/2018, 10:58:12 AM
```
